cleanlab_pinecone_rag_workflow/Cleanlab_Pinecone_RAG.ipynb

```diff
@@ -0,0 +1,5263 @@
+      "Chunk 1 length: 551 characters, 101 tokens\n",
+      "Chunk 2 length: 425 characters, 93 tokens\n",
+      "Chunk 3 length: 479 characters, 98 tokens\n",
+      "Chunk 4 length: 432 characters, 94 tokens\n",
+      "Chunk 5 length: 461 characters, 99 tokens\n",
+      "Chunk 6 length: 452 characters, 96 tokens\n",
+      "Chunk 7 length: 456 characters, 96 tokens\n",
+      "Chunk 8 length: 459 characters, 94 tokens\n",
+      "Chunk 9 length: 496 characters, 99 tokens\n",
+      "Chunk 10 length: 500 characters, 96 tokens\n",
+      "Chunk 11 length: 470 characters, 93 tokens\n",
+      "Chunk 12 length: 439 characters, 93 tokens\n",
+      "Chunk 13 length: 434 characters, 96 tokens\n",
+      "Chunk 14 length: 401 characters, 114 tokens\n",
+      "Chunk 15 length: 397 characters, 103 tokens\n",
+      "Chunk 16 length: 408 characters, 98 tokens\n",
+      "Chunk 17 length: 431 characters, 98 tokens\n",
+      "Chunk 18 length: 454 characters, 103 tokens\n",
+      "Chunk 19 length: 480 characters, 98 tokens\n",
+      "Chunk 20 length: 478 characters, 101 tokens\n",
+      "Chunk 21 length: 461 characters, 97 tokens\n",
+      "Chunk 22 length: 472 characters, 100 tokens\n",
+      "Chunk 23 length: 461 characters, 101 tokens\n",
+      "Chunk 24 length: 446 characters, 96 tokens\n",
+      "Chunk 25 length: 449 characters, 97 tokens\n",
+      "Chunk 26 length: 430 characters, 101 tokens\n",
+      "Chunk 27 length: 422 characters, 100 tokens\n",
+      "Chunk 28 length: 439 characters, 100 tokens\n",
+      "Chunk 29 length: 462 characters, 101 tokens\n",
+      "Chunk 30 length: 486 characters, 104 tokens\n",
+      "Chunk 31 length: 419 characters, 103 tokens\n",
+      "Chunk 32 length: 468 characters, 98 tokens\n",
+      "Chunk 33 length: 439 characters, 104 tokens\n",
+      "Chunk 34 length: 434 characters, 104 tokens\n",
+      "Chunk 35 length: 459 characters, 104 tokens\n",
+      "Chunk 36 length: 481 characters, 104 tokens\n",
+      "Chunk 37 length: 474 characters, 98 tokens\n",
+      "Chunk 38 length: 463 characters, 96 tokens\n",
+      "Chunk 39 length: 461 characters, 97 tokens\n",
+      "Chunk 40 length: 469 characters, 98 tokens\n",
+      "Chunk 41 length: 486 characters, 103 tokens\n",
+      "Chunk 42 length: 467 characters, 97 tokens\n",
+      "Chunk 43 length: 420 characters, 97 tokens\n",
+      "Chunk 44 length: 445 characters, 100 tokens\n",
+      "Chunk 45 length: 473 characters, 91 tokens\n",
+      "Chunk 46 length: 480 characters, 124 tokens\n",
+      "Chunk 47 length: 449 characters, 116 tokens\n",
+      "Chunk 48 length: 409 characters, 106 tokens\n",
+      "Chunk 49 length: 404 characters, 114 tokens\n",
+      "Chunk 50 length: 410 characters, 116 tokens\n",
+      "Chunk 51 length: 407 characters, 103 tokens\n",
+      "Chunk 52 length: 419 characters, 111 tokens\n",
+      "Chunk 53 length: 467 characters, 111 tokens\n",
+      "Chunk 54 length: 428
```

Comment:
every single place you use TLM, you need to show why the trust score is valuable.

Here you were supposed to show how when the trust score is too low, we can skip metadata-filtering our retrieval by topic.